### PR TITLE
Remove icon library and use react-icons

### PR DESCRIPTION
**Pull request by arielsvn**
## Issue Number

#543 

## Purpose/Implementation Notes

Removes all usages of [Ionicons](https://ionicons.com/v1/), we needed only <10 icons from them and we were loading the entire font.

Ionicons was replaced with [react-icons](https://github.com/react-icons/react-icons), the good thing about it is that only the used icons will end in the JS file as svg.

According to pagespeed this should reduce another ~1sec from the render time.

## Types of changes

* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Functional tests

All icons replaced, ensure they look the same.

## Checklist

* [x] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)
* [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by arielsvn** [comment]
Forgot to mention this also includes a fix to ensure all urls in the sitemap are valid.

File: src/components/Spinner.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import spinner from '../common/images/spinner.gif';
 export default function Spinner() {
   return (
     <div className={styles.spinner}>
-      <img src={spinner} width="59" height="54"/>
+      <img src={spinner} width="59" height="54" alt="loading" />
```
Comment (Miserlou) [inline src/components/Spinner.js]: inline style attributes >:|

Comment (arielsvn) [reply src/components/Spinner.js]: Seems that this [is recommended](https://stackoverflow.com/a/6784879/763705), browsers use these attributes as "hints for the rendering".

Comment (arielsvn) [reply src/components/Spinner.js]: took me a while to find that

**Pull request merged**